TeaLeaf/jni/Application.mk

```diff
@@ -1,4 +1,4 @@
 APP_ABI := armeabi-v7a
-APP_PLATFORM := android-21
+APP_PLATFORM := android-14
```

Comment:
this will make sure its builded with all libraries to support from api 14 onwards